src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -551,6 +551,16 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const renderRoomTypeToggle = () => (
+    <div className="toggle-room">
+      <h4 className="italic input-header">set room un-clickable</h4>
+      <label id={"type"} className="switch">
+        <input type="checkbox" id={"type"} name={"type"} ref={register} />
+        <span className="slider round"></span>
+      </label>
```

Comment:
When nesting labels like this we shouldn't need to set the `id`, and in React, we almost never want to use an `id`, as there is no need to.

Similarly, with basic string props, there is no need to wrap them in `{` `}`.

```suggestion
      <label className="switch">
        <input type="checkbox" name="type" ref={register} />
        <span className="slider round"></span>
      </label>
```

Reply:
As mentioned elsewhere, we don't want `type` to be added to venue config, so this may need to be reworked.